contributors/devel/kubemark-guide.md

```diff
@@ -202,9 +234,18 @@ Because there's no easy way of mocking other managers (e.g. VolumeManager), they
 are not supported in Kubemark (e.g. we can't schedule Pods with volumes in them
 yet).
 
-As the time passes more fakes will probably be plugged into HollowNodes, but
-it's crucial to make it as simple as possible to allow running a big number of
-Hollows on a single core.
+We currently plan to extend kubemark along the following directions:
+- As you would have noticed at places above, we aim to make kubemark more structured
+  and easy to run across various providers without having to tweak the setup scripts,
+  using a well-defined kubemark-provider API.
+- Allow kubemark to run on various distros (GCI, debian, redhat, etc) for any
+  given provider.
+- Make Kubemark performance on ci-tests mimic real cluster ci-tests on various metrics
+  and realizing this goal through measurable objectives (like kubemark metrics should
+  vary no more than X% with real metrics).
+- Improve logging of CI-test metrics by packing them into well-structured artifacts
```

Comment:
Aggregated metrics such as API call latencies, scheduling call latencies, percentiles for CPU/mem usage in density and load tests, etc are dumped to `build-log.txt` towards the end of the test.
It would be better to have them written into structured artifacts for use later. Updated this in the doc.

Also, why "Improve" -> "improve"? Followed the same capitalizing convention throughout.